src/user-management/interfaces/refresh-and-seal-session-data.interface.ts

```diff
@@ -16,14 +18,15 @@ export enum RefreshAndSealSessionDataFailureReason {
   ORGANIZATION_NOT_AUTHORIZED = 'organization_not_authorized',
 }
 
+// TODO: These should be renamed since it's possible to have an unsealed session
 type RefreshAndSealSessionDataFailedResponse = {
   authenticated: false;
   reason: RefreshAndSealSessionDataFailureReason;
 };
 
 type RefreshAndSealSessionDataSuccessResponse = {
   authenticated: true;
-  sealedSession: string;
+  session: string | AuthenticationResponse;
```

Comment:
Breaking change?

Reply:
Bah, you're right. I think we'll need a `sealedSession` and `session` to not break this.